doc/source/rllib/rllib-learner.rst

```diff
@@ -0,0 +1,459 @@
+    an algorithm with a custom `Learner` to leverage this API for other algorithms.
+
+    .. list-table::
+       :header-rows: 1
+       :widths: 60 60
+
+       * - Algorithm
+         - Supported Framework
+       * - **PPO**
+         - |pytorch| |tensorflow|
+       * - **Impala**
+         - |pytorch| |tensorflow|
+       * - **APPO**
+         - |pytorch| |tensorflow|
+
+
+Basic Usage
+===========
+
+Use the `LearnerGroup` utility to interact with multiple learners. 
+
+Construction
+------------
+
+If you enable the `RLModule`
+and `Learner` APIs via the RLlib algorithm config, then `Algorithm` constructs a `LearnerGroup` for you, but if you’re using these APIs standalone, you can construct the `LearnerGroup` as follows.
+
+.. testcode::
+    :hide:
+    :skipif: True
+
+    # imports for the examples
+
+    import numpy as np
+    import gymnasium as gym
+    import ray
+    from ray.rllib.algorithms.ppo.torch.ppo_torch_rl_module import PPOTorchRLModule
+    from ray.rllib.algorithms.ppo.torch.ppo_torch_learner import PPOTorchLearner
+    from ray.rllib.algorithms.ppo.ppo_catalog import PPOCatalog
+    from ray.rllib.algorithms.ppo.ppo_learner import PPOLearnerHyperparameters
+    from ray.rllib.core.rl_module.rl_module import SingleAgentRLModuleSpec
+    from ray.rllib.core.learner.learner import FrameworkHyperparameters, LearnerSpec
+    from ray.rllib.core.learner.learner_group import LearnerGroup
+    from ray.rllib.core.learner.scaling_config import LearnerGroupScalingConfig
+
+
+.. tab-set::
+    
+    .. tab-item:: Contstructing a LearnerGroup
+
+
+        .. testcode::
+            :skipif: True
+
+            env = gym.make("CartPole-v1")
+
+            module_spec = SingleAgentRLModuleSpec(
+                            module_class=PPOTorchRLModule,
+                            observation_space=env.observation_space,
+                            action_space=env.action_space,
+                            model_config_dict={},
+                            catalog_class=PPOCatalog
+                        )
+
+            hparams = PPOLearnerHyperparameters(
+                use_kl_loss=True, 
+                kl_coeff=0.01,
+                kl_target=0.05, 
+                clip_param=0.2, 
+                vf_clip_param=0.2, 
+                entropy_coeff=0.05,
+                vf_loss_coeff=0.5
+            )
+
+            scaling_config = LearnerGroupScalingConfig(num_workers=1)
+
+            learner_spec = LearnerSpec(
+                                learner_class=PPOTorchLearner,
+                                module_spec=module_spec,
+                                learner_group_scaling_config=scaling_config,
+                                learner_hyperparameters=hparams,
+                                framework_hyperparameters=FrameworkHyperparameters(),
+                            )
+
+            learner_group = LearnerGroup(learner_spec)
```

Comment:
This would become more useful if it would include the necessary imports so it would become copy-pastable

Reply:
Its going to add a lot of cognitive load to the guide. Maybe instead we put all of this in a file later then come back to it. We can do that when we update the examples.